recipes/rodeo/meta.yaml

```diff
@@ -0,0 +1,89 @@
+package:
+  name: rodeo
+  version: "0.4.4"
```

Comment:
Please template version.  See other feedstocks for example.  Here's one: https://github.com/conda-forge/affine-feedstock/blob/843017788bed7d4b6ee6ae89f91eabfecb23a7ef/recipe/meta.yaml